Explorer/Assets/DCL/PerformanceAndDiagnostics/Analytics/DecoratorBased/ChatMessagesBusAnalyticsDecorator.cs

```diff
@@ -65,19 +66,30 @@ public void Send(ChatChannel channel, string message, ChatMessageOrigin origin,
             analytics.Track(AnalyticsEvents.UI.MESSAGE_SENT, jsonObject);
         }
 
-        private bool  CheckIfIsMention(string message)
+        private bool CheckIfIsMention(string message, out JsonArray mentions)
         {
+            mentions = new JsonArray();
```

Comment:
does it make sense to cache it (create once in the constructor and then `.Clear())`?